mw-server/db/scripts/testdata/testdata.go

```diff
@@ -0,0 +1,188 @@
+package testdata
+
+import (
+	"database/sql"
+	"github.com/gin-gonic/gin"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	_ "github.com/lib/pq"
+	db "mwserver/db/sqlc"
+	"mwserver/util"
+	"net/http"
+	"strings"
+)
+
+var testData = `
+INSERT INTO "users" ("uuid", "name", "email", "description", "image_url", "is_mentor", "firebase_id")
```

Comment:
why not to move it to sql?

Reply:
I wouldn't do that

Reply:
Create sql file for this ticket i\and convert it with sqlc generate to golang code. In this case it will be much more safe

With you approach you will be able to check this script only in runtime, string is not the best format for commands :)